AlfrescoApp/View Controllers/Take Continous Photos View Controller/GalleryPhotosModel.swift

```diff
@@ -179,15 +168,30 @@ protocol GalleryPhotosDelegate: class {
     
     //MARK: Utils
     
+    func defaultNameWithDate(photo: CameraPhoto, index: Int) -> String {
+        let stringIndex = String(index + 1)
+        return imagesName + "_" + Utility.dateFormatter().string(from: Date()) + "_" + stringIndex
+    }
+    
     func shouldShowAlertCellularUpload() -> Bool {
         if let connectivityManager = ConnectivityManager.shared() {
-            if connectivityManager.isOnCellular && selectedPhotos() > 8 {
+            if connectivityManager.isOnCellular && photosSelectedSizeMB() >= 25.0 {
```

Comment:
Could we use the constant that's already in place for size limits, or if not available define one?